client/verta/tests/test_artifacts.py

```diff
@@ -5,16 +5,14 @@
 import hashlib
 import os
 import pickle
-import shutil
 import tempfile
 import zipfile
 
-import requests
-
-from verta._internal_utils import _artifact_utils
-from verta._internal_utils import _utils
-
-from . import utils
+from verta._internal_utils import (
+    _artifact_utils,
+    _file_utils,
+    _utils,
+)
 
 
 class TestUtils:
```

Comment:
I'd like to see some test of the collision avoidance behavior of `download_file` and `download_zipped_dir`.

Reply:
Is this what you'd have in mind? 95bd655c257bd5b6cf2ecd8ace64e7c617b05b00